cmd/root.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright © 2024 NAME HERE <EMAIL ADDRESS>
```

Comment:
NAME HERE and EMAIL ADDRESS i needs to be discussed

Reply:
yup

Reply:
@UjjwalMahar we will discuss later but can you review the functionality?